### DISCUSSION TITLE
Support regular expressions or partial matching for repository names plz.

### DISCUSSION THREAD

**Discussion by maybe-why-not** [community/community#11606] [Other Feature Feedback, Questions, & Ideas] [answered]
Support regular expressions or partial matching for repository names plz.

**Comment by twpayne** [comment]
+1 came here to say this.

My use case: I want to search for use of a library outside forks of that library. The search I want is something like:

    k8s.io/kubernetes/test/e2e NOT repo:*/kubernetes

**Reply by look** [reply]
Would this search be simpler if you could indicate whether or not you want the results in a network of forks? Something like this: `k8s.io/kubernetes/test/e2e NOT is:network(kubernetes/kubernetes)`

I agree searching for repositories by regular expression is useful but in your specific example you could exclude results that aren't forks simply because the repository is named `kubernetes`.

**Reply by twpayne** [reply]
> Would this search be simpler if you could indicate whether or not you want the results in a network of forks?

Yes, that would be perfect.

**Comment by look** [comment, marked as answer]
Thanks for the suggestion. This is on our roadmap, so stay tuned.

**Reply by vchudnov-g** [reply]
Any updates on this?

**Reply by gaston-haro** [reply]
2 years in roadmap
